src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -20,4 +23,30 @@ public LineResponse saveLine(LineRequest request) {
         Line persistLine = lineRepository.save(request.toLine());
         return LineResponse.of(persistLine);
     }
+
+    @Transactional(readOnly = true)
+    public List<LineResponse> findLines() {
+        return lineRepository.findAll().stream()
+                .map(line -> LineResponse.of(line))
```

Comment:
```suggestion
                .map(LineResponse::of)
```
표현도 가능합니다.
알고 계실거 같은데, 혹시몰라 남겼어요